src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -5,4 +5,25 @@ class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    this.adParams = JSON.parse(this.creativeData.adParameters);
+    console.log("Ad params: ", this.adParams);
+    var paramObject = this.adParams[0];
```

Comment:
See comment in design doc, I don't this this format for the adparams quite makes sense

Reply:
spoke through